flow360/component/simulation/models/volume_models.py

```diff
@@ -0,0 +1,128 @@
+from typing import Optional
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.models.material import Air, MaterialType
+from flow360.component.simulation.models.solver_numerics import (
+    NavierStokesSolver,
+    TransitionModelSolver,
+    TurbulenceModelSolverType,
+)
+
+
+class VolumeModelBase(Flow360BaseModel):
+    """
+    Base class for equation models
+
+    """
+
+    volumes: EntityList[Union[GenericVolume, str]] = pd.Field()
+    material: MaterialType = pd.Field()
+    initial_conditions: Optional[dict] = pd.Field(None)
+
+
+class FluidDynamicsAddOnBase(Flow360BaseModel):
+    volumes: EntityList[Union[GenericVolume, str]] = pd.Field()
+
+
+class FluidDynamics(VolumeModelBase):
+    """
+    General FluidDynamics volume model that contains all the common fields every fluid dynamics zone should have.
+    """
+
+    navier_stokes_solver: Optional[NavierStokesSolver] = pd.Field(None)
+    turbulence_model_solver: Optional[TurbulenceModelSolverType] = pd.Field(None)
+    transition_model_solver: Optional[TransitionModelSolver] = pd.Field(None)
+
+    material: MaterialTypes = pd.Field(Air())
```

Comment:
is user supposed to change it? Does our solver support it?

Reply:
For now Air is the only fluid material model exposed. Users can change reference_viscosity and reference_temperature in Sutherland law